data/human/human missions.txt

```diff
@@ -9341,3 +9342,799 @@ mission "Timothy Radrickson 7: Grave"
+			choice
+				`	"Hi, Randalin. It's wonderful to meet you."`
+					goto hirand
+				`	(Nod at the child.)`
+					goto onlyyou
+				`	"She was with you for the battle?"`
+			`	Ildico shrugs. "Who would I trust with her? Who could I trust with her? Only you, <first>, and you were elsewhere."`
+				goto onlyyou
+
+			label hirand
+			`	The little girl lifts a chubby fist up to you.`
+			choice
+				`	(Fist bump her.)`
+					goto bump
+				`	(Shake her fist.)`
+			`	You awkwardly take her fist in your hand and shake it up and down. Randalin giggles, and babbles something incoherent.`
+				goto words
+			
+			label bump
+			`	Her small fist bumps against yours and then she gives you an easy grin and babbles something incoherent.`
+
+			label words
+			`	"She's not quite got the hang of words yet. But they'll come," Ildico says approvingly.`
+			
+			label onlyyou
+			`	Ilidco puts a hand on Randalin's shoulder and draws her close. She turns her attention back to the battlefield.`
+			`	"This is only the beginning, <first>." Her eyes blaze as she says, "House Drifa will rise again. You'll see. You'll be there."`
+			`	She glances back toward her mulling forces. "Thanks again for this, it means a lot that I can count on you, but I have to work to do. Until next time."`
+			`	Ildico returns to her group, barking orders and gesturing at the rubble and leaving you to your own devices.`
+
+
+mission "Ice Queen: News"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		near "Arneb" 10
+	to offer
+		has "Ice Queen 8: done"
+		random < 3
+	on offer
+		conversation
+			`A news kiosk is doing a segment on the growing threat of pirates around Haven. A certain name catches your eye, and you read:`
+			`	Self-styled "pirate queen" Ildico Drifa continues to expand her territory and reach. While there is some concern this growing centralization of power could result in greater threats to trade and stability in the Far North, most analysts think it is quite unlikely.`
+			`	Professor Grenwich Millstone, foremost authority on the Economy and History of the Northern Pariah worlds, had this to say about Drifa:`
+			`	"It is patently inconceivable that any single magnate could consolidate power on Haven. It is culturally anathema, and economically impractical. Frankly, all this talk of a so-called 'pirate queen' is just sensationalist journalism trying to sell the news. In a few weeks another pirate captain will be the new boogieman, and then another."`
+			`	The segment ends, and the news kiosk automatically scrolls down to the next article.`
+				decline
+
+
+mission "Skadenga: Blood of Surtr"
+	landing
+	source "Windblain"
+	to offer
+		has "Ice Queen 6: done"
+		has "outfit: Blood of Surtr"
+	on offer
+		conversation
+			`As your ship descends through Windblain's cloudy atmosphere, you consider landing near Hrithfjall and asking the Ondurdis about the Blood of Surtr.`
```

Comment:
Correct me if I'm wrong, but it seems that you don't actually need to learn about the Skadenga or Ondurdis to get this mission.

Reply:
You're right.

Reply:
Thinking about this. I could just make it so that doing the Skadegna missions is a prerequisite for these.